tests/ballerina-unit-test/src/test/resources/test-src/expressions/conversion/native-conversion.bal

```diff
@@ -661,23 +661,6 @@ function testStructToMapWithRefTypeArray () returns (map<any>, int)|error {
     }
 }
 
-type StructWithDefaults record {
```

Comment:
Can you try this test by making all the fields optional? Then this should be convertible right?

Reply:
Yes that's right but the test case was to test converting to a record with default values.
I added a new test case anyway with 70ac28e